test/e2e/network/netpol/kubemanager.go

```diff
@@ -63,12 +63,14 @@ func (k *kubeManager) initializeCluster(model *Model) error {
 		for _, pod := range ns.Pods {
 			framework.Logf("creating/updating pod %s/%s", ns.Name, pod.Name)
 
-			kubePod, err := k.createPod(pod.KubePod())
+			thePod := pod.KubePod()
```

Comment:
is this diff needed?

Reply:
earlier i had broke this out for mutation but its not needed

Reply:
fixd